.gitignore

```diff
@@ -133,3 +133,5 @@ $RECYCLE.BIN/
 _NCrunch*
 
 .vscode
```

Comment:
this is not useful.  I used JetBrain's rider IDE to implement this feature so I needed to add couple of things to the .gitignore